platforms/web/src/index.tsx

```diff
@@ -1,3 +1,5 @@
+import 'core-js/es/array/flat-map';
+import 'core-js/es/object/from-entries';
```

Comment:
@langemike do you think we can import it conditionally based on the environment variable (APP_LEGACY_BROWSER)?

Reply:
I would not limit this env-var only to the conditional imports, which makes the approach more close like I originally intended, but I think we should keep it simple for now (at least) and make it more flexible if use-cases of demand for this increases.

Reply:
My main concern here is the "polyfills" file we additionally get when we add Chrome legacy support. Even if we don't need it.

Reply:
I revised the approach. Legacy support is now opt-in by providing `APP_LEGACY_BUILD=1` during `yarn build`.

A `polyfills.js` is still created because some dependencies have imported `core-js` internally.

Without legacy support (smaller files):
```
build/public/manifest.json                           0.49 kB │ gzip:   0.29 kB
build/public/index.html                              2.97 kB │ gzip:   0.81 kB
build/public/assets/default_avatar-D3fd0lZl.png     26.96 kB
build/public/assets/style-xyq8hHtg.css             164.70 kB │ gzip:  29.50 kB
build/public/assets/polyfills-BTIFuIxg.js          108.89 kB │ gzip:  43.26 kB │ map:   676.28 kB
build/public/assets/react-Dl1qj1q7.js              143.24 kB │ gzip:  46.11 kB │ map:   351.28 kB
build/public/assets/inplayer-Caq7sMti.js           313.25 kB │ gzip:  92.39 kB │ map:   708.10 kB
build/public/assets/index-BJFB1YLe.js              342.80 kB │ gzip: 103.81 kB │ map: 1,260.17 kB
build/public/assets/vendor-oOs1bjTA.js           1,615.64 kB │ gzip: 431.98 kB │ map: 4,926.55 kB
```

With legacy support (bigger files):
build/public/manifest.json                           0.49 kB │ gzip:   0.29 kB
build/public/index.html                              2.97 kB │ gzip:   0.81 kB
build/public/assets/default_avatar-D3fd0lZl.png     26.96 kB
build/public/assets/style-xyq8hHtg.css             164.70 kB │ gzip:  29.50 kB
build/public/assets/polyfills-DtCB2TJL.js          108.89 kB │ gzip:  43.26 kB │ map:   676.28 kB
build/public/assets/react-Ba7Lyyxn.js              143.28 kB │ gzip:  46.12 kB │ map:   351.34 kB
build/public/assets/inplayer-CHHsr38C.js           313.25 kB │ gzip:  92.39 kB │ map:   708.10 kB
build/public/assets/index-CuUwpo7s.js              343.22 kB │ gzip: 104.00 kB │ map: 1,260.34 kB
build/public/assets/vendor-DQoT3ipD.js           1,615.76 kB │ gzip: 432.01 kB │ map: 4,926.75 kB

Reply:
For some reason they are smaller with APP_LEGACY_BUILD, but the approach itself looks good 👍 Thank you.

Reply:
I think the file listings are switched in the comment above. The added polyfills make the polyfills file larger when `APP_LEGACY_BUILD` is enabled.